packages/core/lib/segments/plugins/plugin.js

```diff
@@ -1,10 +1,17 @@
 var http = require('http');
```

Comment:
if a customer has enabled tracing of all http clients by doing `AWSXRay.captureHTTPsGlobal(require('http'))`, it would trace the calls to the metadata service too right?

Reply:
This is handled later on by:
```js
var httpReq = http.__request ? http.__request : http.request;
```

Which ensures we're using the non-instrumented HTTP client.